src/NHSD.BuyingCatalogue.Solutions.API/ViewModels/SolutionDashboardAuthorityResult.cs

```diff
@@ -0,0 +1,40 @@
+using System.Linq;
+using Newtonsoft.Json;
+using NHSD.BuyingCatalogue.Infrastructure;
+using NHSD.BuyingCatalogue.Solutions.Contracts;
+
+namespace NHSD.BuyingCatalogue.Solutions.API.ViewModels
+{
+    public sealed class SolutionDashboardAuthorityResult
+    {
+        public string Id { get; }
+
+        public string Name { get; }
+
+        [JsonProperty("sections")]
+        public SolutionDashboardAuthoritySections SolutionDashboardAuthoritySections { get; }
+
+        public SolutionDashboardAuthorityResult(ISolution solution)
+        {
+            if (solution != null)
+            {
+                Id = solution.Id;
+                Name = solution.Name;
+                SolutionDashboardAuthoritySections = new SolutionDashboardAuthoritySections(solution);
+            }
+        }
+    }
+
+    public sealed class SolutionDashboardAuthoritySections
+    {
+        [JsonProperty("capabilities")]
+        public DashboardSection Capabilities { get; }
+
+        public SolutionDashboardAuthoritySections(ISolution solution)
+        {
+            solution = solution.ThrowIfNull();
```

Comment:
Like the extension but could we ensure we match the parameter name e.g. nameof(solution)